ivy/functional/ivy/gradients.py

```diff
@@ -86,7 +88,11 @@ def unset_with_grads():
 @to_native_arrays_and_back
 @handle_out_argument
 @handle_nestable
-def variable(x: Union[ivy.Array, ivy.NativeArray]) -> ivy.Variable:
+def variable(
+    x: Union[ivy.Array, ivy.NativeArray],
+    *,
+    out: Optional[ivy.Variable] = None
+) -> ivy.Variable:
```

Comment:
we actually are not using the `ivy.Variable` class anymore, this will eventually be removed. This should instead by `ivy.Array`. We then simply have a property `is_variable` on this array class.